MetaData/data/cross_sections.json

```diff
@@ -117,17 +121,55 @@
+    "GJet_Pt-40toInf_DoubleEMEnriched_MGG-80toInf_TuneCP5_13TeV_Pythia8" : { "xs" : 878.1,  "br" : 1.0,   "kf" : 1.0, "itype":2 },
+    "GJet_Pt-20toInf_DoubleEMEnriched_MGG-40to80_TuneCP5_13TeV_Pythia8"  : { "xs" : 3186.0, "br" : 1.0, "kf" : 1.0, "itype":3 },
+    "GJets_HT-40To100_TuneCUETP8M1_13TeV-madgraphMLM-pythia8" :  { "xs" : 23100.0, "itype" : 4 },
+    "GJets_HT-100To200_TuneCUETP8M1_13TeV-madgraphMLM-pythia8" : { "xs" : 9110.0,  "itype" : 5 },
+    "GJets_HT-200To400_TuneCUETP8M1_13TeV-madgraphMLM-pythia8" : { "xs" : 2280.0,  "itype" : 6 },
+    "GJets_HT-400To600_TuneCUETP8M1_13TeV-madgraphMLM-pythia8" : { "xs" :273.0 , "itype" : 7 },
+    "GJets_HT-600ToInf_TuneCUETP8M1_13TeV-madgraphMLM-pythia8" : { "xs" :94.5 , "itype" : 8 },
+    "GJets_HT-40To100_TuneCP5_13TeV-madgraphMLM-pythia8" :  { "xs" : 18640.0, "itype" : 4 },
+    "GJets_HT-100To200_TuneCP5_13TeV-madgraphMLM-pythia8" : { "xs" : 8631.0,  "itype" : 5 },
+    "GJets_HT-200To400_TuneCP5_13TeV-madgraphMLM-pythia8" : { "xs" : 2185.0,  "itype" : 6 },
+    "GJets_HT-400To600_TuneCP5_13TeV-madgraphMLM-pythia8" : { "xs" :257.7 , "itype" : 7 },
+    "GJets_HT-600ToInf_TuneCP5_13TeV-madgraphMLM-pythia8" : { "xs" :85.4 , "itype" : 8 },
+
+    "DiPhotonJetsBox_M40_80-Sherpa" : { "xs" : 303.2, "itype":10 },
+    "DiPhotonJetsBox_M40_80-sherpa" : { "xs" : 303.2, "itype":10 },
+    "DiPhotonJetsBox_MGG-80toInf_13TeV-Sherpa" : { "xs" : 84.4, "itype":10 },
+    "DiPhotonJetsBox_MGG-80toInf_13TeV-sherpa" : { "xs" : 84.4, "itype":10 },
+    "DiPhotonJets_MGG-80toInf_13TeV_amcatnloFXFX_pythia8" : { "xs" : 135.1 },
+    "DiPhotonJetsBox2BJets_MGG-80toInf_13TeV-Sherpa" : {"xs" : 0.494},
+    "DiPhotonJetsBox1BJet_MGG-80toInf_13TeV-Sherpa" : {"xs" : 0.8674276},
+
+    "QCD_Pt-30to40_DoubleEMEnriched_MGG-80toInf_TuneCUETP8M1_13TeV_Pythia8" : {"xs" : 22110.0 , "br" : 1.0,"itype":30 },
+    "QCD_Pt-40toInf_DoubleEMEnriched_MGG-80toInf_TuneCUETP8M1_13TeV_Pythia8" : {"xs" : 113400.0 , "br" : 1.0,"itype":32 },
+    "QCD_Pt-30toInf_DoubleEMEnriched_MGG-40to80_TuneCUETP8M1_13TeV_Pythia8" : {"xs" : 260500.0 , "br" : 1.0,"itype":31 },      
+    "QCD_Pt-30to40_DoubleEMEnriched_MGG-80toInf_TuneCP5_13TeV_Pythia8" : {"xs" : 24810.0 , "br" : 1.0,"itype":30 },
+    "QCD_Pt-40toInf_DoubleEMEnriched_MGG-80toInf_TuneCP5_13TeV_Pythia8" : {"xs" : 118100.0 , "br" : 1.0,"itype":32 },
+    "QCD_Pt-30toInf_DoubleEMEnriched_MGG-40to80_TuneCP5_13TeV_Pythia8" : {"xs" : 241400.0 , "br" : 1.0,"itype":31 },
+    "QCD_Pt-120to170_EMEnriched_TuneCP5_13TeV_pythia8" : {"xs" : 118100.0 , "br" : 1.0,"itype":32 },
+    "QCD_Pt-170to300_EMEnriched_TuneCP5_13TeV_pythia8" : {"xs" : 118100.0 , "br" : 1.0,"itype":32 },
+    "QCD_Pt-300toInf_EMEnriched_TuneCP5_13TeV_pythia8" : {"xs" : 118100.0 , "br" : 1.0,"itype":32 },
+    "QCD_Pt-50to80_EMEnriched_TuneCP5_13TeV_pythia8" : {"xs" : 118100.0 , "br" : 1.0,"itype":32 },
+    "QCD_Pt-80to120_EMEnriched_TuneCP5_13TeV_pythia8" : {"xs" : 118100.0 , "br" : 1.0,"itype":32 },
```

Comment:
Xsec for QCD with different phase space show the same value which is strange. Could you please check them?